### DISCUSSION TITLE
Welcome to Ciphey Discussions!

### DISCUSSION THREAD

**Discussion by bee-san** [bee-san/Ciphey#559] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BlackHawk0** [comment]
Am getting an erro after i have finished the installation process.
![image](https://user-images.githubusercontent.com/46535202/108834814-ab7a2e00-75df-11eb-8d3e-7cf2fa56d9f2.png)
